lib/update/clusterconfig/phases/services.go

```diff
@@ -0,0 +1,99 @@
+package phases
+
+import (
+	"context"
+	"net"
+
+	"github.com/gravitational/gravity/lib/fsm"
+	"github.com/gravitational/gravity/lib/network/ipallocator"
+	"github.com/gravitational/gravity/lib/utils"
+
+	"github.com/gravitational/trace"
+	log "github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+// NewServices returns a new services step implementation
```

Comment:
Nit: Would be helpful to add a bit more details in the godoc about what this phase does with services.